src/Game.js

```diff
@@ -0,0 +1,63 @@
+// eslint-disable-next-line
+import sampleSize from 'lodash.samplesize';
+
+const attackBot = (enemy, currentPlayer) => {
+  const row = sampleSize([0, 1, 2, 3, 4, 5, 6, 7, 8, 9], 1);
+  const col = sampleSize([0, 1, 2, 3, 4, 5, 6, 7, 8, 9], 1);
+  if (enemy.board.grid[row][col].hit === false) {
+    currentPlayer.attack(enemy, row, col);
+    document.getElementById(`Com${row}${col}`).style.backgroundImage = `url('./img/${enemy.board.grid[row][col].img}.png')`;
+    if (enemy.board.grid[row][col].ship == null) {
+      // eslint-disable-next-line no-use-before-define
+      changeTurn(enemy, currentPlayer);
```

Comment:
`changeTurn` also seems to be calling `attackBot`. Why not create another function to prevent this?